src/Mailer/Transport/SimpleQueueTransport.php

```diff
@@ -49,6 +49,12 @@ public function send(Email $email) {
 			'template' => $email->template(), //template() gives 2 values - template and layout
 			'viewVars' => [$email->viewVars()]
 		];
+		
+		foreach ($settings as $setting => $value) {
+			if (empty($value) || array_key_exists(0, $value) && empty($value[0])) {
```

Comment:
probably because https://github.com/dereuromark/cakephp-queue/blob/master/tests/bootstrap.php#L80-L82 but when using smtp i get

```
Error: [InvalidArgumentException] Reply-To requires only 1 email address
```